cmd/ebrelayer/relayer/cosmos.go

```diff
@@ -239,17 +245,16 @@ func MessageProcessed(message types.CosmosMsg, prophecyClaims []types.ProphecyCl
 func (sub CosmosSub) Replay(fromBlock int64, toBlock int64, ethFromBlock int64, ethToBlock int64) {
 	ProphecyClaims := sub.getAllProphecyClaim(ethFromBlock, ethToBlock)
 
-	fmt.Printf("found out %d prophecy claims I sent from %d to %d block", len(ProphecyClaims), ethFromBlock, ethToBlock)
+	log.Printf("found out %d prophecy claims I sent from %d to %d block\n", len(ProphecyClaims), ethFromBlock, ethToBlock)
```

Comment:
Why not use the new logger here?

Reply:
for the replay scripts, the log not collected and analyzed by tools. so json format not needed. it is better for people to read in console.